src/Coders.cpp

```diff
@@ -10,10 +10,22 @@ Coders::Coders(std::string userName_, std::string discordNickname_, std::string
     ,firecodeLink(firecodeLink_)
     ,group(group_){}
 
+std::string Coders::toString()
+{
+    std::string resultPrint = "";
+    resultPrint += userName;
+    resultPrint += " "+ discordNickname;
+    resultPrint += " "+ githubLink;
+    resultPrint += " "+ firecodeLink + '\n';
+    resultPrint += " "+ group + '\n';
+    return resultPrint;
```

Comment:
it's ok. However string concatenation causes memory allocation. Put everything into a string when you define it, like this:
```cpp
return userName + " " + discordNickname + ...;
```